gen/calc.py

```diff
@@ -1105,6 +1108,8 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'mesos_container_log_sink': 'logrotate',
         'mesos_max_completed_tasks_per_framework': '',
         'mesos_recovery_timeout': '24hrs',
+        'mesos_seccomp_enabled': 'false',
```

Comment:
So by default we will NOT enable this feature even a default seccomp profile is included in DC/OS?

Reply:
Yes, correct. By default, `linux/seccomp` isolator is disabled.